src/main/java/net/apnic/rdap/authority/RDAPAuthorityConfiguration.java

```diff
@@ -0,0 +1,128 @@
+package net.apnic.rdap.authority;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.annotation.PostConstruct;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties
+public class RDAPAuthorityConfiguration
+{
+    private static final Logger LOGGER =
+        Logger.getLogger(RDAPAuthorityConfiguration.class.getName());
+
+    public static class AuthorityConfig
+    {
+        private List<String> aliases;
+        private String name;
+        private List<String> servers;
+
+        public AuthorityConfig()
+        {
+        }
+
+        public List<String> getAliases()
+        {
+            return aliases;
+        }
+
+        public String getName()
+        {
+            return name;
+        }
+
+        public List<String> getServers()
+        {
+            return servers;
+        }
+
+        public void setAliases(List<String> aliases)
+        {
+            this.aliases = aliases;
+        }
+
+        public void setName(String name)
+        {
+            this.name = name;
+        }
+
+        public void setServers(List<String> servers)
+        {
+            this.servers = servers;
+        }
+    }
+
+    private RDAPAuthorityStore authorityStore = new RDAPAuthorityStore();
```

Comment:
Have a look now. Reason I did it this way was to use it in the init method. Possibly still be a better way?